server/test/smart-scheduling.test.ts

```diff
@@ -70,23 +71,174 @@ describe("GET /smart-scheduling/schedules/states/:state.ndjson", () => {
     const data = ndjsonParse(res.body);
     expect(data).toHaveLength(1);
   });
+
+  it("shows available with `has-availability` extension", async () => {
+    let location = await createLocation(TestLocation);
+    await updateAvailability(location.id, {
```

Comment:
nit: For the sake of clarity, maybe create all of these locations (and updating their availability) by looping over an array?

Reply:
813bd079b088cb5316d2cab7b58b5d60cb7d6154 is indeed much nicer. I should make use of `it.each()` more. :)

Reply:
Ah yeah, very nice!